corehq/apps/userreports/es/data_source.py

```diff
@@ -219,29 +220,33 @@ def _get_total_aggregated_results(self):
 
     @method_decorator(catch_and_raise_exceptions)
     def get_total_row(self):
-        total_results = self._get_total_aggregated_results()
+        def _clean_total_row(aggregations, aggregation_name):
+            agg = getattr(aggregations, aggregation_name)
+            if hasattr(agg, 'value'):
+                return agg.value
+            elif hasattr(agg, 'doc_count'):
+                return agg.doc_count
```

Comment:
just curious what is the separation for the two attributes? does ES return different attr names based on the agg?